EDEngineer/Resources/Data/entryData.json

```diff
@@ -2638,7 +2638,29 @@
     "ValueCr": 4000,
     "FormattedName": "employeeexpenses",
     "Kind": "OdysseyIngredient",
-    "OriginDetails": ["Planetary Settlement"],
+    "OriginDetails": [ "Planetary Settlement" ],
+    "Group": "Data",
+    "SettlementType": [],
+    "BuildingType": [],
+    "ContainerType": []
+  },
+  {
+    "Name": "Gene Sequencing Data",
+    "ValueCr": null,

```

Comment:
could this be set to -1? null value would be for when ValueCr is irrelevant to the ingredient (eg pre-odyssey ingredients) while -1 would be invalid / missing price

Reply:
Done